lib/build-cron.js

```diff
@@ -46,14 +46,7 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
           jobs.push(logCleanUpJob);
 
           // Bind job events etc.
-          jobs.forEach((job) => {
-            const handlers = getEventHandlers(job, collection);
-
-            job.on(cmaster.EVENTS.TICK_STARTED, handlers.onTickStarted);
-            job.on(cmaster.EVENTS.TICK_COMPLETE, handlers.onTickComplete);
-            job.on(cmaster.EVENTS.TIME_WARNING, handlers.onTimeWarning);
-            job.on(cmaster.EVENTS.OVERLAPPING_CALL, handlers.onOverlappingCall);
-          });
+          jobs.forEach((job) => instance.attachEventHandlers(job));
```

Comment:
Technically, since we don't rely on prototype/this nonsense, you could just do `jobs.forEach(attachEventsHandlers);` :)

Also, @j-burlison I don't have perms to merge on this repo for whatever reason.